packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -633,6 +702,14 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
     payload?: ClientEventPayload;
     options?: SubmitClientEventOptions;
   }): Promise<any> {
+    this.logger.log(
+      CALL_DIAGNOSTIC_LOG_IDENTIFIER,
+      'CallDiagnosticMetrics: @buildClientEventFetchRequestOptions. Building request options object for fetch()...',
```

Comment:
why ...?

Reply:
I'm guessing there was an issue that Catalin was seeing with a lack of logs. I trust there was a good reason but there is no Jira that describes why the logs were added.

Reply:
No I didn't mean why log is added, I meant why there are three dots?

Reply:
There were other logs that Catalin added ending in "..." so I followed that "convention"